src/test/java/seedu/dukduk/DeadlineTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.duke;
+
+import org.junit.jupiter.api.Test;
+import dukduk.Deadline;
+import java.time.LocalDateTime;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void testToString() {
```

Comment:
You can consider using a more specific function name e.g testToString_Success() to state if it is a positive or negative test.